templates/daemonset.yaml

```diff
@@ -17,18 +17,19 @@ spec:
   template:
     metadata:
       annotations:
+      {{- if .Values.slave.podAnnotationAppArmor.enabled }}
         container.apparmor.security.beta.kubernetes.io/netdata-slave: unconfined
```

Comment:
This should still produce an error if you enable the AppArmor annotation and try to deploy it.  The default name of the container in the pod will be `netdata`, but the container name given here is `netdata-slave`.  It's necessary to use the *container's* name here, not the pod's name.

Reply:
Oops, good spot - I've set it to the chart name in 5bf8a11483cca54ba0b9deda277af49c9a90e8d4